cmd/celestia/rpc.go

```diff
@@ -440,8 +441,8 @@ func parseNamespace(param string) (namespace.ID, error) {
 			return nil, fmt.Errorf("error decoding namespace ID: %w", err)
 		}
 	}
-	// if the namespace ID is 8 bytes, add v0 share + namespace prefix and zero pad
-	if len(nID) == 8 {
+	// if the namespace ID is <= 10 bytes, add v0 share + namespace prefix and zero pad
```

Comment:
[question] does the RPC CLI let user's provide a namespace version? If not, is support for that being tracked somewhere? It's not urgent b/c there is only one namespace version (`0`) but I think we may want to expose it as an optional CLI flag b/c it impacts how namespaces are constructed here

Reply:
@distractedm1nd

Reply:
It's not tracked and not urgent until the next namespaceID versions are out. The CLI is here for providing the easiest way to interact/play with the RPC so I wouldn't add it until it is absolutely necessary